models/model.js

```diff
@@ -12,9 +12,15 @@ exports.fetchArticleById = async (id) => {
     [id.article_id]
   );
 
+  const { rows: comments } = await db.query(
```

Comment:
we should be creating a comment count inside the first query using an aggregate function and joining two tables together rather than querying the database twice here - think about what psql function we can use and what two tables need to be joined